config/atlantis-common.toml

```diff
@@ -0,0 +1,35 @@
+#Should Islands Generate?
+islandsOn = true
+#Should Volcanoes Generate?
+volcanoesOn = true
+#Should Glowstone Crysts Generate?
+glowstoneCrystsOn = true
+#Minimum Crab Spawn Height
+#Range: -64 ~ 320
+minCrabSpawnHeight = 100
+#Maximum Crab Spawn Height
+#Range: -64 ~ 320
+maxCrabSpawnHeight = 125
+#Rate at which Magma Accelerates you
+#Range: 0.0 ~ 100.0
+magmaAcceleration = 0.1
+#Max Distance that the Push Bubble Columns from Bubble Magma Blocks can go
+#Range: 0 ~ 30
+maxDistanceOfPushBubbleColumn = 15
+#Max Magma Velocity 
+#Range: 0.0 ~ 100.0
```

Comment:
Velocity or threshold? Or it's the same?

Reply:
I don't remember; I generated the default configs, copied them and modified